code/controllers/subsystem/SStimer.dm

```diff
@@ -550,6 +544,21 @@ SUBSYSTEM_DEF(timer)
 	prev = null
 	bucket_list[bucket_pos] = src
 
+/**
+ * Returns debug information about timer
+ */
+/datum/timedevent/proc/getTimerInfo()
```

Comment:
If it generates and sets timer `name` wouldn't it be better to have proc named:
`getTimerName`

Reply:
Well, that proc just caches crucial debug information in a `name` variable so it can be viewed easily through VV. It may be misleading though if renamed, since its purpose is debug helping.
![изображение](https://github.com/ParadiseSS13/Paradise/assets/73733747/3a5e97fa-fc89-451d-bb20-1ed62269753f)